src/main/java/gui/DialogBox.java

```diff
@@ -0,0 +1,62 @@
+package gui;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+
+/**
+ * An example of a custom control using FXML.
+ * This control represents a dialog box consisting of an ImageView to represent the speaker's face and a label
```

Comment:
Even though it's exactly 110 in length (which still complies with the coding standard), perhaps would be a good idea to break it into two lines. It's too long to be seen when people review the code.

Reply:
Got it!